packages/sdk/src/utils/handlePanic.ts

```diff
@@ -62,6 +62,7 @@ ${chalk.dim(`Learn more: ${link('https://pris.ly/d/telemetry')}`)}
       console.log(`\n${chalk.bold(`We successfully received the error report id: ${reportId}`)}`)
       console.log(`\n${chalk.bold('Thanks a lot for your help! 🙏')}`)
     } catch (error) {
+      // TODO: we should report why we're not able to submit an error report
```

Comment:
What do you mean here?

Reply:
It was a reminder to push and release this PR: https://github.com/prisma/prisma/pull/13257

Reply:
Ok then we can remove the comment and merge main / rebase?

Reply:
```suggestion
```
You can commit with `[skip ci]`